packages/menu/src/Menu.story.tsx

```diff
@@ -70,7 +70,7 @@ function Controlled() {
           <MenuItem href="http://mongodb.design" size={size}>
             I am a link!
           </MenuItem>
-          <MenuSeparator />
+          <MenuSeparator margin={0} />
```

Comment:
This breaks a long-standing pattern we've intentionally avoided of directly adding CSS properties as component props (with the exception of `z-index`). Is there any reason we aren't setting a default margin through CSS and allowing a className override here instead of exposing a margin prop?

Reply:
Maybe I misunderstood your comment here? https://jira.mongodb.org/browse/PD-1873

Reply:
I feel like we wouldn't want other CSS properties to be overridden, but maybe that's overly restrictive

Reply:
I suppose, if we are going to give the ability to override the spacing, it should be through className. The component probably doesn't need it at all though.

Reply:
The alternative, in cases where spacing is needed, would be to wrap the separator in a div, and apply the spacing to that. I think it's better to just support that on the separator itself

Reply:
Since it's not something that we support now, it might be best to wait until there's a use-case for the override to add an API for one.

Reply:
Updated to use `className`